src/main/scala/io/keen/client/scala/EventStore.scala

```diff
@@ -0,0 +1,24 @@
+package io.keen.client.scala
+
+import java.io.IOException
+
+import scala.collection.concurrent.TrieMap
+import scala.collection.mutable.ListBuffer
+
+trait EventStore {
+
+  @throws(classOf[IOException])
+  def store(projectId: String,
+    eventCollection: String,
+    event: String): Long
+
+  @throws(classOf[IOException])
+  def get(handle: Long): String
+
+  @throws(classOf[IOException])
+  def remove(handle: Long): Unit
+
+  @throws(classOf[IOException])
+  def getHandles(projectId: String): TrieMap[String, ListBuffer[Long]]
```

Comment:
We can be more general than `TrieMap` and `ListBuffer` at this level in the hierarchy.